src/mme-app/utils/mmeCommonUtils.cpp

```diff
@@ -70,7 +71,9 @@ uint32_t MmeCommonUtils::allocateMtmsi()
 
 	while(1)
 	{
-		tmsi = rand() % 10000;
+		std::default_random_engine generator(time(NULL));
```

Comment:
may be generator should be static variable. We may not earn much by initializing generator again n again in each loop. You can put it outside while loop